src/ws.rs

```diff
@@ -96,8 +96,19 @@ pub(crate) fn convert_ws_message(message: Option<Message>) -> Result<Option<Even
         // simd-json::serde::from_str may leave an &mut str in a non-UTF state on failure.
         // The below is safe as we have taken ownership of the inner `String`, and don't
         // access it as a `str`/`String` or return it if failure occurs.
-        Some(Message::Text(mut payload)) =>
-            unsafe { crate::json::from_str(payload.as_mut_str()) }.map(Some)?,
+        Some(Message::Text(mut payload)) => {
```

Comment:
You've accidentally included #170 here, but since the merge will already be really messy we should leave it for now to prevent complaints over the weekend.